Umbraco.Docs.Samples.Web/Property-Editors-Add-Values/CreateImageCropperValuesController.cs

```diff
@@ -51,10 +48,16 @@ public ActionResult<string> CreateImageCropperValues()
             // Serialize the image cropper value
             var cropperValue = JsonConvert.SerializeObject(cropper);
 
+            var testCropperAlias = Product.GetModelPropertyType(_publishedSnapshotAccessor, x => x.TestCropper)?.Alias;
+
             // Set the value of the property with alias 'cropper'
-            content.SetValue("testCropper", cropperValue, "en-US");
+            if (content != null && !string.IsNullOrWhiteSpace(testCropperAlias))
```

Comment:
@erikjanwestendorp This sample is a little confusing, perhaps group the 2 approaches together and add a comment to explain?

Reply:
@Jeavon Sorry for the delay on this one. I just updated the PR.